app/src/main/java/com/framgia/soundcloud/screen/home/offline/OfflinePresenter.java

```diff
@@ -21,4 +37,30 @@ public void onStop() {
     public void setViewModel(OfflineContract.ViewModel viewModel) {
         mViewModel = viewModel;
     }
+
+    @Override
+    public void getTracks() {
+        ContentResolver resolver = mContext.getContentResolver();
+        Uri uri = MediaStore.Audio.Media.EXTERNAL_CONTENT_URI;
+        Cursor cursor = resolver.query(uri, null, null, null, null);
```

Comment:
WARNING: [Recycle] This `Cursor` should be freed up after use with `#close()`
 Many resources, such as TypedArrays, VelocityTrackers, etc., should be recycled (with a `recycle()` call) after use. This lint check looks for missing `recycle()` calls.